samples-android/ButtonClicker/jni/ButtonClickerNativeActivity.cpp

```diff
@@ -105,15 +105,14 @@ void Engine::InitGooglePlayGameServices() {
  *
  */
 void Engine::OnAuthActionStarted(gpg::AuthOperation op) {
+  startup_mutex_.lock();
   if (!initialized_resources_) return;
+
   ndk_helper::JNIHelper::GetInstance()->RunOnUiThread([this, op]() {
     EnableUI(false);
-    authorizing_ = true;
     if (op == gpg::AuthOperation::SIGN_IN) {
-      LOGI("Signing in to GPG");
```

Comment:
removed a couple of LOG messages from sample to make it clean.  when UI is up, it is displayed UI page at the bottom in status_text. so we could make it  a little clean in logcat